darwin/exporter/formats/coco.py

```diff
@@ -0,0 +1,185 @@
+import json
+from datetime import date
+from pathlib import Path
+from typing import Generator, List
+
+import numpy as np
+
+import darwin.datatypes as dt
+
+
+class NumpyEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, np.integer):
+            return int(obj)
+        elif isinstance(obj, np.floating):
+            return float(obj)
+        elif isinstance(obj, np.ndarray):
+            return obj.tolist()
```

Comment:
Will a numpy array of numpy integers return a list of numpy integers or a list of python integers? Just checking